### PR TITLE
feat(connector): [NOMUPAY] Add template code

### PR DESCRIPTION
**Pull request by sumanmaji4**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Template connector code for Nomupay payout

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [ ] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Comment by ImSagnik007** [comment]
resolve the failing checks and add openapi changes as well

File: config/development.toml [hunk 1]
```diff
@@ -237,6 +238,7 @@ multisafepay.base_url = "https://testapi.multisafepay.com/"
 netcetera.base_url = "https://{{merchant_endpoint_prefix}}.3ds-server.prev.netcetera-cloud-payment.ch"
 nexinets.base_url = "https://apitest.payengine.de/v1"
 nexixpay.base_url = "https://xpaysandbox.nexigroup.com/api/phoenix-0.0/psp/api/v1"
+nomupay.base_url = "https://payout-api.sandbox.nomupay.com"
```
Comment (cookieg13) [inline config/development.toml:241]: please fix the alphabetical here

File: config/docker_compose.toml [hunk 2]
```diff
@@ -247,6 +248,7 @@ cards = [
     "netcetera",
     "nexinets",
     "nexixpay",
+    "nomupay",
```
Comment (cookieg13) [inline config/docker_compose.toml:251]: same

File: crates/hyperswitch_interfaces/src/configs.rs [hunk 3]
```diff
@@ -53,6 +53,7 @@ pub struct Connectors {
     pub netcetera: ConnectorParams,
     pub nexinets: ConnectorParams,
     pub nexixpay: ConnectorParams,
+    pub nomupay: ConnectorParams,
```
Comment (cookieg13) [inline crates/hyperswitch_interfaces/src/configs.rs:56]: same

File: loadtest/config/development.toml [hunk 4]
```diff
@@ -213,6 +214,7 @@ cards = [
     "netcetera",
     "nexinets",
     "nexixpay",
+    "nomupay",
```
Comment (cookieg13) [inline loadtest/config/development.toml:217]: same

**Review by cookieg13 — CHANGES_REQUESTED** [review]

**Review by cookieg13 — APPROVED** [review]

**Review by deepanshu-iiitu — APPROVED** [review]

**Review by sai-harsha-vardhan — APPROVED** [review]

**Review by sai-harsha-vardhan — APPROVED** [review]

**Review by deepanshu-iiitu — APPROVED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Review by cookieg13 — DISMISSED** [review]

**Review by deepanshu-iiitu — DISMISSED** [review]

File: crates/router/tests/connectors/nomupay.rs [hunk 5]
```diff
@@ -0,0 +1,427 @@
...
+            types::Connector::Plaid,
+            api::GetToken::Connector,
+            None,
+        )
+        // api::ConnectorData {
+        //     connector: Box::new(Nomupay::new()),
+        //     connector_name: types::Connector::Nomupay,
+        //     get_token: types::api::GetToken::Connector,
+        //     merchant_connector_id: None,
+        // }
```
Comment (SanchithHegde) [inline crates/router/tests/connectors/nomupay.rs:25]: Remove commented code.

**Review by SanchithHegde — APPROVED** [review]

**Review by cookieg13 — DISMISSED** [review]

**Review by sai-harsha-vardhan — APPROVED** [review]

**Review by deepanshu-iiitu — APPROVED** [review]

**Review by cookieg13 — APPROVED** [review]

**Review by sai-harsha-vardhan — APPROVED** [review]

**Review by dracarys18 — APPROVED** [review]

**Review by ArjunKarthik — APPROVED** [review]

**Pull request merged**